src/tests/PokemonDetails.test.js

```diff
@@ -8,43 +9,41 @@ describe('Testing structure PokemonDetails.js', () => {
   it('Testa se as informações detalhadas do Pokémon são mostradas na tela', () => {
     const pikachu = pokemons[0];
 
-    const { getByText, history, queryByRole, container } = renderWithRouter(<App />);
+    const { history } = renderWithRouter(<App />);
 
     history.push(`/pokemons/${pikachu.id}`);
 
-    const pikachuName = getByText(/Pikachu details/i);
-    const link = queryByRole('link', { name: 'More details' });
-    const h2 = container.querySelectorAll('h2');
-    const textH2 = h2[1];
-    const getAllP = document.querySelectorAll('p');
-    const summaryP = getAllP[3];
+    const pikachuName = screen.getByText(/Pikachu details/i);
+    const link = screen.queryByRole('link', { name: 'More details' });
+    const SummaryTitle = screen.getByRole('heading', { name: /Summary/i });
+    const SummaryContent = screen.getByText(/This intelligent Pokémon/i);
```

Comment:
```suggestion
    const summaryTitle = screen.getByRole('heading', { name: /Summary/i });
    const summaryContent = screen.getByText(/This intelligent Pokémon/i);
```

Aqui as variáveis poderiam ser com letras minúsculas?

Reply:
Ok!